src/hashutil.h

```diff
@@ -12,11 +12,9 @@
 namespace hashing {
 // See Martin Dietzfelbinger, "Universal hashing and k-wise independent random
 // variables via integer arithmetic without primes".
-/*
```

Comment:
It is fine to re-enable it!
Just FYI: I disabled TwoIndependentMultiplyShift before so that I don't use it by accident.

Reply:
I totally agree. I also the same hash function (TwoIndependentMultiplyShift) in my repository for all filters.